### ISSUE TITLE
Add posibility to ignore build assets on rebase command

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/.github#28]
When trying to rebase a PR with committed js bundles it would be great if one could easily tell that those should be ignored during the rebase as a compile command would fix the conflict afterwards.

**Comment by juliushaertl** [comment]
https://github.com/cirrus-actions/rebase/blob/master/entrypoint.sh#L100 is where the handling would need to take palce

**Comment by Pytal** [comment]
Would be superseded by https://github.com/orgs/nextcloud/projects/47#card-65665432

**Comment by max-nextcloud** [comment]
> https://github.com/cirrus-actions/rebase/blob/master/entrypoint.sh#L100 is where the handling would need to take palce

I don't think we even get to that point. I think the rebase action that we are using errs out when it github says the commit is not rebaseable:
https://github.com/cirrus-actions/rebase/blob/master/entrypoint.sh#L45

**Comment by max-nextcloud** [comment]
I don't know if jobs can be combined like that but we could:
1. `git checkout HEAD~ js && git commit --amend js --no-edit`
2. `/rebase`
3. `/compile amend /`

First line only works for a single commit though. We could apply it to all commits with git filter branch or a rebase --exec...
But frankly this sounds fairly complicated and like we were asking for trouble.

**Comment by MichaIng** [comment]
Probably it's possible to somehow set a flag to 1 an 2 to prevent CI?
